packages/fluentui/react-northstar/src/components/Avatar/AvatarImage.tsx

```diff
@@ -0,0 +1,120 @@
+import { Accessibility, AccessibilityAttributes, imageBehavior, ImageBehaviorProps } from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useUnhandledProps,
+  useAccessibility,
+  useFluentContext,
+  useStyles,
+  useTelemetry,
+} from '@fluentui/react-bindings';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { UIComponentProps, commonPropTypes } from '../../utils';
+import { FluentComponentStaticProps } from '../../types';
+import { imageClassName } from '../Image/Image';
+
+export interface AvatarImageProps extends UIComponentProps, ImageBehaviorProps {
+  /** Alternative text. */
+  alt?: string;
+
+  'aria-label'?: AccessibilityAttributes['aria-label'];
+
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<ImageBehaviorProps>;
+
+  /** An Avatarimage may be formatted to appear inline with text as an avatar. */
+  avatar?: boolean;
```

Comment:
Could be worth considering deprecating this from `Image` in this PR

Reply:
and all `avatar` related styles there

Reply:
I think it probably is going a little out of the scope of this PR and probably a breaking change we don't want, `Image` can still be used as avatar if consumer wants, not necessary only inside `Avatar`. WDYT @pompomon @jurokapsiar @layershifter

Reply:
Let's not break usage now, but let's create a followup task to remove it

Reply:
You can put `@deprecated` tag even now

Reply:
yup, not looking to remove, only deprecating

Reply:
Please write the deprecation description in the PR description like we discussed in open hours

Reply:
Added